docs/guides/advanced/errors.md

```diff
@@ -0,0 +1,139 @@
+---
+nav: Error handling
+description: Handle exceptions and display error messages to users.
+---
+
+# Handling errors
+
+Dealing with errors is a common task when building a Pack. These errors can come from outside sources such as an API, or you may have to create them yourself when users pass invalid inputs.
+
+
+## Fetcher errors
+
+When the fetcher receives a response from the API indicating there was a problem it throws an exception that needs to be handled. More information about these errors can be found in the [Fetching remote data guide][fetcher].
```

Comment:
Well, technically you can let it throw because Coda will catch fetcher errors and convert them to generic error messages for the end user.

Reply:
Good point, fixed.